src/SFA.DAS.TrackProgress.Api/Controllers/TrackProgressController.cs

```diff
@@ -0,0 +1,13 @@
+using Microsoft.AspNetCore.Mvc;
+
+namespace SFA.DAS.TrackProgressApi.Controllers;
+
+[ApiController]
+[Route("[controller]")]
+public class TrackProgressController : ControllerBase
+{
+    [HttpGet(Name = "TrackProgress")]
```

Comment:
Does this override the Route["controller"] so are they both needed?

Reply:
You're right, we can remove the class level `Route`